### ISSUE TITLE
[chromecast] Could not resolve type id 'DEVICE_ADDED'

### ISSUE THREAD

**Issue by chriscolden** [openhab/openhab2-addons#3719]
Getting the following in my logs. I've not noticed anything broken as such. Ie the binding does seem to be working.

```
11:28:59.778 [WARN ] [su.litvak.chromecast.api.v2.Channel  ] - Error while handling
org.codehaus.jackson.map.JsonMappingException: Could not resolve type id 'DEVICE_ADDED' into a subtype of [simple type, class su.litvak.chromecast.api.v2.StandardResponse]
 at [Source: N/A; line: -1, column: -1]
        at org.codehaus.jackson.map.JsonMappingException.from(JsonMappingException.java:163) ~[229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.deser.StdDeserializationContext.unknownTypeException(StdDeserializationContext.java:273) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.jsontype.impl.TypeDeserializerBase._findDeserializer(TypeDeserializerBase.java:117) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:74) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:89) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.deser.StdDeserializerProvider$WrappedDeserializer.deserialize(StdDeserializerProvider.java:494) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.ObjectMapper._readValue(ObjectMapper.java:2695) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1990) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.EventListenerHolder.deliverEvent(EventListenerHolder.java:56) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.Channel.notifyListenersOfSpontaneousEvent(Channel.java:440) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.Channel.access$600(Channel.java:50) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.Channel$ReadThread.run(Channel.java:186) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
```

Also seeing the following, not sure if its related
```
11:29:35.529 [WARN ] [su.litvak.chromecast.api.v2.Channel  ] - Error while reading
su.litvak.chromecast.api.v2.ChromeCastException: Remote socket closed
        at su.litvak.chromecast.api.v2.Channel.read(Channel.java:413) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.Channel.access$200(Channel.java:50) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
        at su.litvak.chromecast.api.v2.Channel$ReadThread.run(Channel.java:132) [229:org.openhab.binding.chromecast:2.4.0.201806282119]
11:29:35.567 [WARN ] [su.litvak.chromecast.api.v2.Channel  ] -  <--  null payload in message
```

openHAB 2.4.0 Build #1307
openjdk version "1.8.0_152"
Raspbian Stretch

**Comment by wborn** [comment]
When everything works properly perhaps the log levels of the Chromecast library need to be adjusted. WDYT @vitalidze? 

The openHAB Chromecast binding uses chromecast-java-api-v2 v0.10.1. There don't seem to be any fixes for this in the latest release (v0.10.2). It would be nice to see a v0.10.3 because of the better IOException handling.

**Comment by vitalidze** [comment]
@wborn I think this must be updated for sure. I have expressed my opinion on the way to update in the PR that you have sent me recently https://github.com/vitalidze/chromecast-java-api-v2/pull/101
